packages/graphql/src/translate/create-connect-and-params.ts

```diff
@@ -159,8 +159,13 @@ function createConnectAndParams({
             subquery.push(`\tWHERE ${whereStrs.join(" AND ")}`);
         }
 
-        const preAuth = [...(!fromCreate ? [parentNode] : []), relatedNode].reduce(
-            (result: Res, node, i) => {
+        const nodeMatrix: { node: Node; name: string }[] = [
+            ...(!fromCreate ? [{ node: parentNode, name: parentVar }] : []),
+            { node: relatedNode, name: nodeName },
+        ];
```

Comment:
**suggestion (non-blocking)**
Maybe it would be a bit more readable by using `.push` here?

```suggestion
        const nodeMatrix: Array<{ node: Node; name: string }> = [ { node: relatedNode, name: nodeName }];
        if (fromCreate) nodeMatrix.push({node: parentNode, name: parentVar});
```